### ISSUE TITLE
apex link to label

### ISSUE THREAD

**Issue by mshanemc** [forcedotcom/salesforcedx-vscode#850]
**Is your feature request related to a problem? Please describe.**
I can't remember what all my labels are

**Describe the solution you'd like**
When an apex class or a LWC references a label, and my local source has that label, link to it and offer the definition

**Describe alternatives you've considered**
searching and opening labels in a new tab, using extraordinarily verbose label names

**Additional context**
example: here, I'd want to be able to peek at the label.  It's just xml.
<img width="535" alt="screen shot 2019-01-07 at 7 46 21 am" src="https://user-images.githubusercontent.com/4261788/50771470-5be61600-1250-11e9-9f85-54430ffd9b57.png">

<img width="653" alt="screen shot 2019-01-07 at 7 47 46 am" src="https://user-images.githubusercontent.com/4261788/50771540-8b951e00-1250-11e9-8685-171fe389ce65.png">

**Comment by RenegadeCoder** [comment]
Yes! I would **love** to stop seeing `External string does not exist` in my problems view as well
